iap_watcher/startup_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Please rename this file to `start_iap_watcher.sh` to be more consistent with our naming elsewhere.

Reply:
Done.